worker/subprojects/libsrtp2.wrap

```diff
@@ -1,8 +1,8 @@
 [wrap-file]
-directory = libsrtp-4c9f0956f2933ac0650208d69c8d897625ba6301
-source_url = https://github.com/versatica/libsrtp/archive/4c9f0956f2933ac0650208d69c8d897625ba6301.zip
-source_filename = libsrtp-4c9f0956f2933ac0650208d69c8d897625ba6301.zip
-source_hash = 4f3af61e26df398569605fc4bcf377587ca2d8bd34b2b4bf9cdb9590cadbd662
+directory = libsrtp-37d88ecac103f4e2a5336099c64b155ba4ca3b0c
+source_url = https://github.com/versatica/libsrtp/archive/37d88ecac103f4e2a5336099c64b155ba4ca3b0c.zip
+source_filename = libsrtp-37d88ecac103f4e2a5336099c64b155ba4ca3b0c.zip
+source_hash = edf8b2c7a27ed6b6aac64c8d6a165379d3a0042bd420706b4a1825acc182441d
```

Comment:
Why pulling from a fork rather than upstream?

Also FYI GitHub doesn't guarantee the hash of ZIP files from random revisions to not change over time. It is only guaranteed for releases.

Reply:
> Why pulling from a fork rather than upstream?

This is a mistake, I thought I was taking it from libsrtp repo directly.

> Also FYI GitHub doesn't guarantee the hash of ZIP files from random revisions to not change over time. It is only guaranteed for releases.

That's right.., libsrtp version 3 is to be released sometime this year.., I'll ask for the timeline of the 2.5 release. I may create a PR for the enhancement to be included in 2.5.

Reply:
Why don't we create a GH relaase in our libsrtp fork and point to it in the wrap file?

Reply:
That would be a solution indeed

Reply:
> Also FYI GitHub doesn't guarantee the hash of ZIP files from random revisions to not change over time. It is only guaranteed for releases.

I can't read where it says that a zip file for a certai commit hash may change. Indeed the documentation says it won't.

https://docs.github.com/en/repositories/working-with-files/using-files/downloading-source-code-archives#stability-of-source-code-archives

Reply:
> An archive of a commit ID will always have the same file contents whenever it's requested, assuming the commit ID is still in the repository and the repository's name has not changed.

Only contents because:
> The exact compression settings used to generate a zipball or tarball may change over time. The extracted contents won't change if the branch or tag doesn't change, but the outer compressed archive may have a different byte layout. GitHub will give at least six months' notice before changing compression settings.

Reply:
Over a year ago GH changed the compression settings and half of the projects (including mediasoup) got into problems, so this is a real thing. However GH took the problem into consideration hence that new policy. Probably we are good whatever we do here but the only reliable way to go is by having a GH release.

Reply:
Nice, I'll do a release from our fork then as suggested :+1: